src/components/chart/ChargebackChart.tsx

```diff
@@ -58,10 +58,14 @@ export default ({ chartData }) => {
     );
   }
 
+  const cutLabel = (label: string) =>
+    label.length > 25 ? label.substring(0, 25) + '...' : label;
```

Comment:
You can provide "25" into argument of function
 `(label: string, maxLength = 25)`

then you don't have "magic numbers" in your code, and function is more flexible